root/release/edit/information.tt

```diff
@@ -329,7 +329,7 @@
       [% l('The purpose of this field is to add information that usually doesn\'t fit into the strict structural data schema of MusicBrainz (be it due to technical limitations that may be addressed later, or because the information in itself has to be free-text).') %]
     </p>
     <p>
-      [% l('This annotation will be published under an open license (<a href="{url}" target="_blank">CC BY-NC-SA 3.0 </a>) and as such, it should not contain text that you don\'t have the right to release under that license. While you can quote a source to support a point you're making, you should never enter promotional texts or other artist or label-owned texts into the annotation.',
+      [% l('This annotation will be published under an open license (<a href="{url}" target="_blank">CC BY-NC-SA 3.0</a>) and as such, it should not contain text that you don\'t have the right to release under that license. While you can quote a source to support a point you're making, you should never enter promotional texts or other artist or label-owned texts into the annotation.',
```

Comment:
Same issue again with `you're`, the single quote `'` should be escaped as the whole string is delimited with single quotes. It's a bug causing the string to end before `re making`.